### PR TITLE
Force transforming collection assets is now handled in a more general way

### PR DESCRIPTION
**Pull request by jankrassnigg**
This code is now also executed on TransformAll not only when a scene is exported.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -12,7 +12,7 @@ indent_style = space
 trim_trailing_whitespace = true
 
 # MSVC specific
-cpp_generate_documentation_comments = doxygen_triple_slash
+cpp_generate_documentation_comments = none
```
Comment (Sanakan8472) [inline .editorconfig:15]: Why this change?

**Review by Sanakan8472 — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (jankrassnigg) [reply .editorconfig:15]: Because the default then is that when you type /// VS will auto complete it to /// @brief.
That was also the intention. However, we use /// \brief. So now I always need to modify it again, which is annoying.
So either we agree that using @ is also fine, then it is useful, or we just continue doing it manually. And I didn't have the energy for yet another discussion about these kind of things, I had enough of them the last weeks :D

Comment (Sanakan8472) [reply .editorconfig:15]: Well Ok I have no strong feeling one way or another. We could just use `@`, the end result will be the same.
